src/foraging_gui/Foraging.py

```diff
@@ -3766,7 +3766,21 @@ def _set_default_project(self):
             self.Metadata_dialog.project_info = project_info
             self.Metadata_dialog.ProjectName.addItems([project_name])
         return project_name
-
+    def _empty_old_data(self):
```

Comment:
What is this doing?

Reply:
To clear or initialize certain fields from previous sessions when the new session is started without restarting the GUI. These fields may interfere with the generation of the manifest file if not reset.

Reply:
That doesn't seem related to skipping the manifest generation for ephys sessions. Can you make a dedicated PR